ui/src/components/ClientScopesTable/index.jsx

```diff
@@ -28,6 +30,11 @@ const sorted = pipe(
   map(({ node: { clientId } }) => clientId)
 );
 
+@withStyles({
+  tableText: {
+    fontSize: '0.8125rem',
+  },
+})
```

Comment:
I think this won't be needed after we remove the font-size from the body element in Main.jsx. Same comment applies to the other places where we do this kind of change.